public/components/appManager.js

```diff
@@ -0,0 +1,98 @@
+const { ipcMain, app, BrowserWindow } = require('electron');
+const path = require('path');
+const windowManager = require('./windowManager');
+const registerProtocolProtocol = require('./protocolProtocol').registerProtocolProtocol;
+
+function getFileFromArgs(argv) {
+  if (argv.length >= 2) {
+    const filePath = argv[1];
+    if (path.extname(filePath) === '.netcanvas') {
+      console.log('.netcanvas found in argv', JSON.stringify({ argv }, null, 2));
+      return filePath;
+    }
+  }
+  return null;
+}
+
+const appManager = {
+  openFileWhenReady: null,
+  init: function init() {
+    ipcMain.on('GET_ARGF', (event) => {
+      if (process.platform === 'win32') {
+        const filePath = getFileFromArgs(process.argv);
+        if (filePath) {
+          event.sender.send('OPEN_FILE', filePath);
```

Comment:
These are the result of bringing over the updater functionality from Architect. I thought about removing them, but realised we will implement protocol loading via the file menu in NC at some point, and so decided to leave them.

As far as I know this shouldn't cause an issue, except that the feature doesn't work because nothing is handling the other end (as you pointed out). Do you think I should remove it?